zebrad/tests/common/config.rs

```diff
@@ -15,6 +15,11 @@ use zebrad::{
     config::{SyncSection, TracingSection, ZebradConfig},
 };
 
+/// The environmental variable that contains the cached state path.
+///
+/// TODO: use this in the checkpoint cached state tests as well.
+pub const CACHED_STATE_PATH_VAR: &str = "ZEBRA_CACHED_STATE_PATH";
```

Comment:
Minor: In #4068, I renamed this constant to

```suggestion
pub const ZEBRA_CACHED_STATE_DIR_VAR: &str = "ZEBRA_CACHED_STATE_DIR";
```